cmake/configs/base.cmake

```
@@ -200,27 +201,25 @@ config_string(
 config_string(
     VERSION_MAJOR
     "Major version number for WiredTiger"
-    DEFAULT 10
+    DEFAULT ${WT_VERSION_MAJOR}
 )
 
 config_string(
     VERSION_MINOR
     "Minor version number for WiredTiger"
-    DEFAULT 0
+    DEFAULT ${WT_VERSION_MINOR}
 )
 
 config_string(
     VERSION_PATCH
     "Path version number for WiredTiger"
-    DEFAULT 1
+    DEFAULT ${WT_VERSION_PATCH}
 )
 
-
-string(TIMESTAMP config_date "%Y-%m-%d")
 config_string(
     VERSION_STRING
     "Version string for WiredTiger"
-    DEFAULT "\"WiredTiger ${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH} (${config_date})\""
+    DEFAULT "${WT_VERSION_STRING}"
```

So sorry, I just realized we need to maintain escaped quotes, without it the CMake build will break. The following change will fix the issue: 
```suggestion
    DEFAULT "\"${WT_VERSION_STRING}\""
```